File: app/src/main/java/com/najib/cryptowatcher/DetailCryptoActivity.java
package com.najib.cryptowatcher;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.TextView;

import com.androidnetworking.widget.ANImageView;

public class DetailCryptoActivity extends AppCompatActivity {

    public static final String EXTRA_CRYPTO = "extra_crypto";
    ANImageView logoCrypto;
    TextView symbol,name, price, low, high;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        Crypto crypto = getIntent().getParcelableExtra(EXTRA_CRYPTO);
        logoCrypto = findViewById(R.id.logoCoin);
        symbol = findViewById(R.id.txtSymbolCrypto);
        name = findViewById(R.id.txtNameCrypto);
        price = findViewById(R.id.txtCurrentPrice);
        low = findViewById(R.id.txtLowPrice);
        high = findViewById(R.id.txtHighPrice);

        logoCrypto.setImageUrl(crypto.getImage());
        symbol.setText(crypto.getSymbol());
        name.setText(crypto.getName());
        price.setText(crypto.getCurrentPrice());
        low.setText(crypto.getLowPrice());
        high.setText(crypto.getHighPrice());

        Log.d(DetailCryptoActivity.class.getName(),""+crypto.getName());
    }
}

File: app/src/main/java/com/najib/cryptowatcher/DetailCryptoActivity.kt
package com.najib.cryptowatcher

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.najib.cryptowatcher.databinding.ActivityDetailBinding

class DetailCryptoActivity : AppCompatActivity() {

    private lateinit var binding: ActivityDetailBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityDetailBinding.inflate(layoutInflater)
        setContentView(binding.root)
        val crypto: Crypto? = intent.getParcelableExtra(EXTRA_CRYPTO)

        binding.logoCoin.setImageUrl(crypto?.image)
        binding.txtSymbolCrypto.text = crypto?.symbol
        binding.txtNameCrypto.text = crypto?.name
        binding.txtCurrentPrice.text = crypto?.currentPrice
        binding.txtLowPrice.text = crypto?.lowPrice
        binding.txtHighPrice.text = crypto?.highPrice

        Log.d(DetailCryptoActivity::class.java.name, "" + crypto?.name)
    }

    companion object {
        const val EXTRA_CRYPTO = "extra_crypto"
    }
}